compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEAccessToAcquireRelease.cpp

```diff
@@ -4,88 +4,110 @@
 // See https://llvm.org/LICENSE.txt for license information.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+#include "iree-amd-aie/IR/AMDAIEAttrs.h"
 #include "iree-amd-aie/IR/AMDAIEOps.h"
+#include "iree-amd-aie/Transforms/AMDAIEUtils.h"
 #include "iree-amd-aie/Transforms/Passes.h"
-#include "mlir/IR/IRMapping.h"
-#include "mlir/IR/Iterators.h"
 
 #define DEBUG_TYPE "iree-amdaie-access-to-acquire-release"
 
 namespace mlir::iree_compiler::AMDAIE {
 
 namespace {
 
+/// Some blocks have terminator ops, which must appear as the very last op in
+/// the block. If `block` has a terminator, set the insertion point of
+/// `rewriter` to just before the terminator, ready to create a new penultimate
+/// op in the block. Otherwise, set the insertion point to the very end of the
+/// block.
+void setInsertionToEnd(IRRewriter &rewriter, Block *block) {
+  if (block->back().hasTrait<OpTrait::IsTerminator>())
+    rewriter.setInsertionPoint(block->getTerminator());
+  else
+    rewriter.setInsertionPointToEnd(block);
```

Comment:
```suggestion
  if (block->back().hasTrait<OpTrait::IsTerminator>()) {
    rewriter.setInsertionPoint(block->getTerminator());
  } else {
    rewriter.setInsertionPointToEnd(block);
  }
```
Is this your formatter? You might need to set it on Google code style.

Reply:
No, I don't have any fancy formatter, just missed this one. 

Personally I wish C++ had never allowed dropping braces in the first place, but let's leave that can of worms closed...